terraform/snowflake/modules/policies/variables.tf

```diff
@@ -0,0 +1,21 @@
+variable "okta_integration_name" {
+  description = "The name of the Okta security integration. If null, the odi_okta_only and admin_okta_duo policies will not be created."
+  type        = string
+  default     = null
+}
+
+variable "policies_database_name" {
+  description = "Name of the database to store policies"
+  type        = string
+  default     = "POLICIES"
+}
+
+variable "logging_warehouse_name" {
+  description = "The name of the warehouse to be used by the Sentinel service user."
+  type        = string
+}
+
+variable "logger_role_name" {
+  description = "The name of the role to be granted to the Sentinel service user."
+  type        = string
+}
```

Comment:
I don't see a strong reason for these to be configurable (all of our other functional roles are pretty hard-coded)